workspaces/orchestrator/README.md

```diff
@@ -1,115 +1,53 @@
 
-The orchestrator works harmoniously with other Backstage components such as the Software Catalog, permissions, and plugins as well as others. By leveraging its capabilities, organizations can orchestrate and coordinate developer self-service flows effectively.
-
-## Context
-
-The Backstage Orchestrator plugin aims to provide a better option to Scaffolder, based on workflows to have a more flexible and powerful tool that addresses the need by streamlining and automating processes, allowing developers to focus more on coding and innovation.
+It utilizes [SonataFlow](https://sonataflow.org/), a powerful tool for building cloud-native workflow applications.
 
-The orchestrator relies on [SonataFlow](https://sonataflow.org/), a powerful tool for building cloud-native workflow applications.
-
-The main idea is to keep the same user experience for users, leveraging the UI components, input forms, and flow that Scaffolder provides, this way it should be straightforward for users and transparent no matter whether using Templates or Workflows, both can live together being compatible with integration points.
+The orchestrator works harmoniously with other Backstage components such as the Software Catalog, permissions, and plugins as well as others. By leveraging its capabilities, organizations can orchestrate and coordinate developer self-service flows effectively.
 
-The orchestrator controls the flow orchestrating operations/tasks that may be executed in any external service including Scaffolder Actions, this way it is possible to leverage any existing Action hence Software Templates can be easily migrated to workflows opening the door to extend them to more complex use cases.
+## Architecture Overview
 
-## Capabilities
+The architecture adheres to standard Backstage plugin guidelines and requires the following plugins to be installed for proper functionality:
 
-**Advanced core capabilities**
+1. **`backstage-plugin-orchestrator`**
 
-- Stateful/long-lived
-- Branching and parallelism
-- Error management and compensation
-- Event-driven supporting [CloudEvents](https://cloudevents.io)
-- Audit logging
-- Sub-flows
-- Choreography
-- Timer/timeout control
-- Built-in powerful expression evaluation with JQ
-- Low Code/No code
-- Cloud-native architecture Kubernetes/OpenShift with Operator support
-- OpenAPI / REST built-in integration etc.
+   - Provides the frontend interface.
 
-**Client-side tooling**
+2. **`backstage-plugin-orchestrator-backend`**
+   - Serves as a backend proxy between to SonataFlow.
 
-- Orchestration visualization / graphical editor
-- Integration with service catalog/actions
-- GitHub integration
-- Form generation
-- Runtime monitoring of instances
-- Dashboards
-- Potential custom integrations (user interaction, notifications, etc.)
+In addition to the main plugins, the architecture includes several isolated plugins that are imported by the above plugins:
```

Comment:
doesn't apply anymore?

Reply:
It does, I thought better make it shorter